docs/en/getting-started/get-started-docker.asciidoc

```diff
@@ -288,183 +293,116 @@ Settings (Windows).
-generated for the `kibana_system` user.
-+
---
-ifeval::["{release-state}"=="unreleased"]
-NOTE: Version {version} of {es} has not been released,
-so the sample compose file is not yet available for this version.
-See the {stack-gs-current}/get-started-docker.html[current version] for the latest sample files.
-endif::[]
+[[docker-compose-remove-containers]]
+[discrete]
+==== Remove Docker Compose containers
 
-ifeval::["{release-state}"!="unreleased"]
-["source","yaml",subs=+quotes]
-----
-  kib01:
-    image: docker.elastic.co/kibana/kibana:${VERSION}
-    container_name: kib01
-    depends_on: {"es-node01": {"condition": "service_healthy"}}
-    ports:
-      - 5601:5601
-    environment:
-      SERVERNAME: localhost
-      ELASTICSEARCH_URL: https://es-node01:9200
-      ELASTICSEARCH_HOSTS: https://es-node01:9200
-      ELASTICSEARCH_USERNAME: kibana_system
-      **ELASTICSEARCH_PASSWORD: CHANGEME**
-      ELASTICSEARCH_SSL_CERTIFICATEAUTHORITIES: $CERTS_DIR/ca/ca.crt
-      SERVER_SSL_ENABLED: "true"
-      SERVER_SSL_KEY: $CERTS_DIR/kib01/kib01.key
-      SERVER_SSL_CERTIFICATE: $CERTS_DIR/kib01/kib01.crt
-    volumes:
-      - certs:$CERTS_DIR
-    networks:
-      - elastic
-----
-endif::[]
---
+When you're done experimenting, you can remove the network, containers, and
+volumes:
 
-. Use `docker-compose` to restart the cluster and {kib}:
-+
---
-["source","sh"]
+[source,sh]
 ----
-docker-compose stop
-docker-compose -f elastic-docker-tls.yml up -d
+docker-compose down -v
 ----
---
-
-. Open {kib} to load sample data and interact with the cluster:
-https://localhost:5601.
-+
-NOTE: Because SSL is also enabled for communications between {kib} and client browsers,
-you must access {kib} via the HTTPS protocol.
-
-When you're done experimenting, you can tear down the containers, network, and
-volumes by running `docker-compose -f elastic-docker-tls.yml down -v`.
 
 [discrete]
 [[load-settings-file]]
-==== Loading settings from a file
+==== Load settings from a file
 
-Specifying settings for {es} and {{kib}} directly in the compose file is a convenient way to get started,
-but loading settings from a file is preferable once you get past the experimental stage.
+Specifying settings for {es} and {kib} directly in the Docker Compose file is a
+convenient way to get started, but loading settings from a file is preferable
+after you get past the experimental stage.
 
-For example, to use `es-node01.yml` as the configuration file for the `es-node01` {es} node,
-you can create a bind mount in the volumes section.
+For example, to use a custom `es01.yml` as the configuration file for the `es01`
+{es} node, you can create a bind mount in the `volumes` section for the `es01`
+service.
 
 ["source","yaml"]
 ----
     volumes:
       - data01:/usr/share/elasticsearch/data
       - certs:$CERTS_DIR
-      - ./es-node01.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+      - ./es01.yml:/usr/share/elasticsearch/config/elasticsearch.yml
```

Comment:
The other lines for the snippet should be removed since they refer to the previous version of the `docker-compose.yml` file.

In order to avoid confusion if we'll update it again, what about adding something like this:

```
----
    volumes:
      - ./es01.yml:/usr/share/elasticsearch/config/elasticsearch.yml
      - ...
```

Same in the Kibana snippet.

Reply:
++